### ISSUE TITLE
1.14

### ISSUE THREAD

**Issue by Dangman368** [GalaxyOnVR/MonkeCodegen-and-Utils#3]
MAKE 1.14 MODS

**Comment by CanyonsEcho** [comment]
YOU CANT MOD 1.14

**Comment by kaithub** [comment]
oh really?
![1.54 modded](https://github.com/GalaxyOnVR/MonkeCodegen-and-Utils/assets/140258324/750ef2fe-d905-4141-b5be-4d0ed4e6df10)

**Comment by CanyonsEcho** [comment]
> oh really? ![1.54 modded](https://user-images.githubusercontent.com/140258324/257037358-750ef2fe-d905-4141-b5be-4d0ed4e6df10.png)

yeah really, you cant make any mods work cause they were broken by newer updates like monke codegen stopped working in rainy day

**Comment by kaithub** [comment]
i was able to update the **PATCH** method slightly, i still do need the core mods, correct.

**Comment by starseed12345** [comment]
> i was able to update the **PATCH** method slightly, i still do need the core mods, correct.

anything?
